new/LIT6987PPNames.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6987PPNames" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer with the names of God</title>
+                <title xml:lang="gez" xml:id="t2">ቀዳሚሁ፡ ቃል፡ ውእቱ፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Qadāmihu qāl wǝʾǝtu…</title>
```

Comment:
Delete '...'. Otherwise it looks as if you have not translated the complete title as noted above.